setup.py

```diff
@@ -0,0 +1,31 @@
+import os
+
+from setuptools import setup, find_packages
+
+here = os.path.abspath(os.path.dirname(__file__))
+with open(os.path.join(here, 'README.rst')) as f:
+    readme = f.read()
+
+with open(os.path.join(here, 'VERSION')) as f:
+    version = f.read().strip()
+
+setup(name='casepro.pods.familyconnect',
+      version=version,
+      description=('.'),
```

Comment:
Maybe we should add a description here (just noticed the dummy pod also doesn't have a description, will fix, sorry about that).